tests/baselines/reference/findAllRefs_importType_js.1.baseline.jsonc

```diff
@@ -15,7 +15,7 @@
       "containerName": "",
       "fileName": "/a.js",
       "kind": "local class",
-      "name": "(local class) C",
+      "name": "(local class) C\nmodule C",
```

Comment:
This new result is more correct (it accurately reflects the merge done on the symbol), however the "new" result isn't actually stable - symbol merges are ad-hoc in the checker, and the merge that produces this result is only created when the symbol for the _source file_ is pulled on (as by diagnostic production), but not when the local _class_ is pulled on (you can probably trigger the same inconsistency today by requesting all refs for the file followed by all refs for the class)! This is probably an amazingly subtle bug somewhere in JS checking that's been revealed by this diff, and fixing it is a separate issue. (The fix should involve making the merge no matter which entrypoint requests the symbol.)

Reply:
Opened https://github.com/microsoft/TypeScript/issues/48197 to track the issue.